tests/python/relax/test_frontend_dynamo.py

```diff
@@ -498,9 +513,7 @@ def main(
 
     class Select2(Module):
         def forward(self, input1):
-            result = input1[
-                torch.arange(1),
-            ]
+            result = input1[torch.arange(1),]
```

Comment:
style. Keep it remain unchanged or 
```suggestion
            result = input1[torch.arange(1)]
```